### PR TITLE
(PCP-157) Update and clean-up

### PR DESCRIPTION
**Pull request by parisiale**

### CODE REVIEW

File: pcp/README.md [hunk 1]
```diff
@@ -31,10 +31,11 @@ WebSockets as the underlying wire protocol.
 
  - [pcp-broker][41] provides a pcp-broker implementation in Clojure
  - [cpp-pcp-client][42] is a PCP client library written in C++
- - [pxp-agent][43] is an agent used to run puppet, based on cpp-pcp-client
+ - [pxp-agent][43] is a PXP agent based on cpp-pcp-client, that offers a Puppet module
  - [clj-pcp-common][44] is a library for processing pcp messages written in
  Clojure
- - [clj-pcp-client][45] is a PCP client ibrary written in Clojure *TODO(ploubser) Link is broken since we're mid rename
+ - [clj-pcp-client][45] is a PCP client ibrary written in Clojure
+ - [clj-pxp-puppet][46] is a PXP client library for interacting with the pxp-agent's Puppet module
```
Comment (ploubser) [inline pcp/README.md]: Remove, it's not a public component.

File: pcp/README.md [hunk 2]
```diff
@@ -53,4 +54,5 @@ WebSockets as the underlying wire protocol.
 [42]: https://github.com/puppetlabs/cpp-pcp-client
 [43]: https://github.com/puppetlabs/pxp-agent
 [44]: https://github.com/puppetlabs/clj-pcp-common
-[45]: https://github.com/puppetlabs/clj-pcp-client 
+[45]: https://github.com/puppetlabs/clj-pcp-client
+[46]: https://github.com/puppetlabs/clj-pxp-puppet
```
Comment (ploubser) [inline pcp/README.md]: Same as above

File: pcp/message.md [hunk 3]
```diff
@@ -61,7 +56,9 @@ entry of the envelope.
 
 A message may have zero or multiple debug chunks. Debug chunks are appended to the
 back of the message by either clients or servers, and their [content](#chunk-format)
-is described by the following json-schema.
+is described by the following JSON schema.
+ 
+*TODO(ale):* replace "server" with "broker"
```
Comment (ploubser) [inline pcp/message.md]: Get rid of this TODO?

Comment (parisiale) [inline pcp/message.md]: Ok, it's worth removing them all.

**Pull request merged**